packages/web-components/src/tooltip/tooltip.ts

```diff
@@ -0,0 +1,116 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+
+/**
+ * A Tooltip Custom HTML Element.
+ * @public
+ */
+export class Tooltip extends FASTElement {
+  public elementInternals = this.attachInternals();
+
+  /**
+   * Set the delay for the tooltip
+   */
+  @attr
+  public delay: number = 250;
```

Comment:
This will end up reflecting, which isn't necessarily the worst, but I wonder if there is a way to default this without reflecting the attributes default value?